### ISSUE TITLE
Masonry re-position issue

### ISSUE THREAD

**Issue by ev5unleash** [metafizzy/outlayer#20]
Hi all,

So I'm having an issue where Masonry will only try to fit DIVs next to each other if it has excessive space. For instance, I have 2 divs one with 300px width and another with 700px width in a container of 1000px width. Mathmatically, 300 + 700 = 1000 no? When I try to use masonry on this layout, it will stack DIVs on top of each other instead of next to each other. The only way I've gotten it to work is to increase with width of the container to about 300px more than the required width.

Working but not whitespace: https://dl.dropboxusercontent.com/u/8556362/Capture1.PNG
Not-working, but should: https://dl.dropboxusercontent.com/u/8556362/Capture.PNG

Example Code:

<!DOCTYPE html>
<html>
<head>
    <title></title>
</head>

<style>
    #container {
        margin: 0 auto;
        padding: 0;
        width: 1000px !important; //310 div1 + 650 div 2 + 20 (margins) = 980. Does not work
width: 1300px !important; //Giving it an extra 300px works but obviously not want I'm going for
        height: 500px;
        background: #007733;
    }

    .box {
        background-color: #000000;
        margin: 5px;
        width: 1000px;
        height: 100px;
    }
</style>

<script src="//ajax.googleapis.com/ajax/libs/jquery/1.11.1/jquery.min.js"></script>

<script src="masonry.js"></script>

<script>
    setTimeout(function() {
        var container = document.querySelector("#container");
            window.msnry = new Masonry( container, {
                columnWidth: 0,
                itemSelector: ".box",
                gutterWidth: 0,
                isFitWidth: false,
                isAnimated: true
            });
        },1000);

</script>

<body>

<div id="container">
    <div class="box" style="width: 310px; height: 500px;"></div> 
    <div class="box" style="width: 650px; height: 500px;"></div>
    <div class="box"></div>
    <div class="box"></div>
    <div class="box"></div>
    <div class="box"></div>
    <div class="box"></div>
    <div class="box"></div>

</div>

</body>
</html>

Not sure if I'm doing something wrong or this is actually a bug. Any help would be appreciated.
